src/saveload/mod.rs

```diff
@@ -0,0 +1,13 @@
+//! Save and load entites from various formats with serde
```

Comment:
I would really appreciate if you could add a top-level overview here.

Reply:
It would be better if someone with better writing skills do this